dockerfiles/executor_image/data/registries.conf

```diff
@@ -0,0 +1,21 @@
+# Note that changing the order here may break tests.
```

Comment:
The original file can be found at: https://github.com/containers/podman/blob/main/test/registries.conf